src/client/src/database.rs

```diff
@@ -473,8 +473,8 @@ impl Database {
         }) = &self.ctx.auth_header
         {
             let encoded = BASE64_STANDARD.encode(format!("{username}:{password}"));
-            let value =
-                MetadataValue::from_str(&encoded).context(InvalidTonicMetadataValueSnafu)?;
+            let value = MetadataValue::from_str(&format!("Basic {encoded}"))
```

Comment:
It seems that we also need to update the java, go and rust ingester

Reply:
Does it mean this will be a breaking change?

Reply:
We need to keep it backward compatible. If it's really necessary to add `Basic` prefix, we need to support both cases.

Reply:
PTAL